hiss/application/admin.py

```diff
@@ -69,6 +72,20 @@ def reject(_modeladmin, _request, queryset) -> None:
             instance.save()
 
 
+def export_emails(_modeladmin, _request, queryset):
```

Comment:
Try and add as many type hints as you can to functions. Also, this name conflicts with the other `export_emails` function in `user`.

```suggestion
from django.db.models.query import QuerySet
# ...

def export_application_emails(_modeladmin, _request: HttpRequest, queryset: QuerySet) -> HttpResponse:
```